portfolio/target/maven-archiver/pom.properties

```diff
@@ -1,5 +1,5 @@
 #Generated by Maven
-#Sat Apr 18 00:23:39 PDT 2020
```

Comment:
Can you add the target directory to the .gitignore file so that these aren't tracked? Feel free to let me know if you have any questions on this